test/sealer_images_test.go

```diff
@@ -112,4 +114,65 @@ var _ = Describe("sealer images module", func() {
 			image.DoImageOps("push", pushImageName)
 		})
 	})
+
+	Context("mount and umount a single image", func() {
+		BeforeEach(func() {
+			image.DoImageOps("pull", settings.TestImageName)
+			testhelper.CheckBeTrue(build.CheckIsImageExist(settings.TestImageName))
+		})
+
+		It("start to mount and umount", func() {
+
+			By("start to mount a image")
+			mountCmd := fmt.Sprintf("%s alpha mount %s -d", settings.DefaultSealerBin, settings.TestImageName)
+			testhelper.RunCmdAndCheckResult(mountCmd, 0)
+
+			By("start to umount a image")
+			umountCmd := fmt.Sprintf("%s alpha umount %s -d", settings.DefaultSealerBin, settings.TestImageName)
+			testhelper.RunCmdAndCheckResult(umountCmd, 0)
+
+		})
+	})
+
+	Context("mount and unmount multiple images", func() {
+		BeforeEach(func() {
+			image.DoImageOps("pull", settings.TestImageName)
+			image.DoImageOps("pull", settings.TestK8s120)
+			image.DoImageOps("pull", settings.TestK8s118)
+			testhelper.CheckBeTrue(build.CheckIsImageExist(settings.TestImageName))
+			testhelper.CheckBeTrue(build.CheckIsImageExist(settings.TestK8s120))
+			testhelper.CheckBeTrue(build.CheckIsImageExist(settings.TestK8s118))
+		})
+
+		It("start to mount and umount", func() {
+
+			By("start to mount multiple images")
+			mountCmd := fmt.Sprintf("%s alpha mount %s %s %s -d", settings.DefaultSealerBin, settings.TestImageName, settings.TestK8s120, settings.TestK8s118)
+			testhelper.RunCmdAndCheckResult(mountCmd, 0)
+
+			err := filepath.WalkDir("/var/lib/containers/storage/overlay", func(path string, d fs.DirEntry, err error) error {
```

Comment:
why "/var/lib/containers/storage/overlay" , make it const or get it from imageengine ?